frontend/src/metabase/admin/settings/components/widgets/ModelCachingScheduleWidget/ModelCachingScheduleWidget.styled.tsx

```diff
@@ -24,7 +24,6 @@ export const WidgetContainer = styled.div`
 
 export const StyledSettingSelect = styled(SettingSelect)`
   width: 125px;
-  min-height: 45.5px; // should match SettingInput height
```

Comment:
This doesn't seem to do anything on the new `TextInput` component, so I removed it.